src/client/components/region-banner-component/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
```

Comment:
I see you wanted to have 2 components in the same folder. And it's okay. But in this case you should name the folder same as the parent component. So, it would be `components/region-banner-list` and the Banner component: `components/region-banner-list/item.jsx`